tests/test_onnx_export.py

```diff
@@ -80,12 +80,17 @@ def to_numpy(tensor):
     return tensor.cpu().numpy()
 
 
-def set_layer_scale(module: torch.nn.Module, scale: float):
-    module.fp8_init()
-    module.fp8_meta["scaling_fwd"].scale = torch.ones(
-        2, dtype=torch.float32, device="cuda") / scale
-    module.fp8_meta["scaling_fwd"].scale_inv = torch.ones(
-        2, dtype=torch.float32, device="cuda") * scale
+def set_layer_scale(module: torch.nn.Module, scales: float, num_gemms: int=1):
```

Comment:
You can remove `num_gemms` because we don't need/use it.
Please change the typing hint for `scales` to List.

```suggestion
def set_layer_scale(module: torch.nn.Module, scales: List[float]):
```

Reply:
`num_gemms` is required for the `fp8_init()` call. For LayernormMLP specifically, it is set to 2. (see line below)

Reply:
My bad - I missed that - ignore my comment